test/main.js

```diff
@@ -52,42 +52,41 @@
-                    'spec/config/collections/memorials/detailFields.spec',
-                    'spec/config/collections/memorials/searchResultFields.spec',
+                    'spec/config/collections/library/searchResultFields.spec',
+                    'spec/config/collections/library/detailFields.spec',
                     'spec/config/collections/memorials/searchTypes.spec',
-                    'spec/config/collections/museum/detailFields.spec',
-                    'spec/config/collections/museum/searchResultFields.spec',
+                    'spec/config/collections/memorials/searchResultFields.spec',
+                    'spec/config/collections/memorials/detailFields.spec',
                     'spec/config/collections/museum/searchTypes.spec',
-                    'spec/config/collections/photographs/detailFields.spec',
-                    'spec/config/collections/photographs/searchResultFields.spec',
+                    'spec/config/collections/museum/searchResultFields.spec',
+                    'spec/config/collections/museum/detailFields.spec',
                     'spec/config/collections/photographs/searchTypes.spec',
-                    'spec/config/components.spec',
-                    'spec/config/routes.spec',
-                    'spec/config/settings.spec',
-                    'spec/models/DynamicRecord.spec',
-                    'spec/models/ListPager.spec',
+                    'spec/config/collections/photographs/searchResultFields.spec',
+                    'spec/config/collections/photographs/detailFields.spec',
+                    'spec/config/env/uat.spec',
+                    'spec/config/env/staging.spec',
+                    'spec/config/env/development.spec',
                     'spec/models/ListSorter.spec',
+                    'spec/models/ListPager.spec',
                     'spec/models/ListModeSwitcher.spec',
-                    'spec/services/cachingService.spec',
+                    'spec/models/DynamicRecord.spec',
                     'spec/services/searchService.spec',
                     'spec/services/detailService.spec',
+                    'spec/services/cachingService.spec',
                     'spec/ui/components/controls/ListControlsComponent.spec',
                     'spec/ui/components/display/DisplayComponent.spec',
                     'spec/ui/pages/detail/DetailPage.spec',
                     'spec/ui/pages/error/ErrorPage.spec',
-                    'spec/ui/pages/search/SearchPage.spec',
                     'spec/ui/pages/search/SearchType.spec',
+                    'spec/ui/pages/search/SearchPage.spec',
                     'spec/ui/pages/static/StaticPage.spec',
-                    'spec/util/bind.spec',
+                    'spec/util/safelyParseJson.spec',
                     'spec/util/container.spec',
-                    'spec/util/safelyParseJson.spec'
+                    'spec/util/bind.spec'
```

Comment:
These are the same tests, in a different order.  The line number difference is because the `TODO` comment at the top has been taken away.